apps/emqx_bridge_snowflake/README.md

```diff
@@ -0,0 +1,22 @@
+# EMQX Snowflake Bridge
+
+This application provides connector and action implementations for the EMQX to integrate with Snowflake as part of the EMQX data integration pipelines.
+Users can leverage [EMQX Rule Engine](https://docs.emqx.com/en/enterprise/latest/data-integration/rules.html) to create rules that publish message data to Snowflake.
+
+## Documentation
+
+Refer to [Rules engine](https://docs.emqx.com/en/enterprise/latest/data-integration/rules.html) for the EMQX rules engine introduction.
+
+## Implementation details
+
+At the time of writing, it's not possible to use Snowflake's SDKs which support [Snowpipe Streaming](https://github.com/snowflakedb/snowflake-ingest-java/blob/master/README.md#snowpipe-streaming).  Here, we try to approximate what the SDK does, which is to buffer rows into (Parquet) files before uploading (staging) and ingesting them in bulk using currently undocumented Snowpipe APIs.
```

Comment:
The API is now documented https://docs.snowflake.com/en/user-guide/data-load-snowpipe-rest-apis

Reply:
Yes, that was what I meant to say.  This is a more precise way to phrase it, thanks.